pkg/cassandra/nodetool/client/client.go

```diff
@@ -0,0 +1,89 @@
+package client
```

Comment:
Can we get rid of this sub-package and just absorb its functionality into `pkg/cassandra/nodetool`? I'm not too sure what the difference between these packages is, it seems they are both clients? Can this be folded into a single client? 😄